From 67b14b40a109350409c1f2cae777d48abe1735b4 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Mon, 9 Jul 2018 19:14:16 +0300
Subject: [PATCH 559/767] staging: fsl-dpaa2/eth: Enable debugfs support by
 default

The debugfs counters of the DPAA2 Ethernet driver are useful in
many occasions and bring no runtime overhead, so it makes sense
to enable them by default.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index 2ea561f488c0..e756916c2772 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -39,7 +39,7 @@ config FSL_DPAA2_ETH_USE_ERR_QUEUE
 config FSL_DPAA2_ETH_DEBUGFS
 	depends on DEBUG_FS
 	bool "Enable debugfs support"
-	default n
+	default y
 	---help---
 	  Enable advanced statistics through debugfs interface.
 
-- 
2.17.0

